index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Initial HTML file</title>
+</head>
+<body>
+    <h1 class="myfavheader">Main Heading</h1>
```

Comment:
- [ ] [REQUIRED] This pull request can be improved by updating the title and description. 
* Pull Request should have a descriptive title telling what changes are inside.
* Pull Request should have a summary describing introduced changes in more detail.
e.g Title -> Hello Microverse project and the description -> Mention the things you did in brief detail.